### ISSUE TITLE
Change the algorithm automatic mode

### ISSUE THREAD

**Issue by SOTVM** [bpierre/gtranslate#67]
Good morning / afternoon / night.
forgive me for the mistakes, if any
I do not speak English

Now the program can specify only one direction of translation.(auto > SET lang.)
it would be great if it were possible in automatic mode
set these settings
example:
1) IF highlighted Rus. text - THEN translated into English,
2) IF highlighted text is not Russian. - THEN translated into Russian

This can be corrected in the code?

**Comment by mote0230** [comment]
I want this too, so I created a pull request https://github.com/bpierre/gtranslate/pull/68

**Comment by PerfectSlayer** [comment]
@SOTVM Yes it's a good idea! :+1: 
I already though it should work like that.

@mote0230 I'll have a look to your PR for the end of the week if nobody did it before :wink:

**Comment by SOTVM** [comment]
@PerfectSlayer  
pop-up window with the translation should appear immediately when you hover over a menu item, not the sub-menu.
You do not need much action.
and would be nice to add hot keys "show translation" + "copy to clipboard"
and have the option in the settings to show perevot hover
it's not hard to do?


I will try to explain in pictures  
because if translate my Russian. Then you do not understand anything.
you understand me? you know what I want to explain?
[please see the picture](https://yadi.sk/i/wTU0SvzxrSjeQ)
you understand what I'm trying to say?

**Comment by PerfectSlayer** [comment]
So you want the popup on the main menu level, on « Translate "xxx" » label ?
Adding hotkeys to show translation and copy clipboard are another features. They are not related to algorithm mode, aren't they ?

**Comment by SOTVM** [comment]
@PerfectSlayer 
Yes, this is not related to the issue  "Change the algorithm automatic mode #67 "


it is translated correctly?
You can understand me?
**please help make the issue as well**

1) show the translation in a popup window when the cursor is on the menu, not a submenu
add hot keys
2) show the translation in a popup window
3) copy the translation to the clipboard
4) and be able to change these settings, add-on options

**Comment by SOTVM** [comment]
@PerfectSlayer 
вот так я хотел - https://addons.mozilla.org/Ru/firefox/addon/s3google-translator
как я не видел раньше это дополнение?
лучшая из того что я пробовал
